SP2022 - Settlers (#21)
sourcebots/smallpeice-rules: specs.tex

\label{spec:arena}

\begin{enumerate}
  \item The arena floor is an \SI{8.4}{m} $\times$ \SI{8.4}{m} rectangle. The
        tolerance of these two dimensions is $\pm$ \SI{250}{mm}.
  \item The floor of the arena is carpeted.
  \item The layout of the arena is given in Figure~\ref{fig:arena}. This
        figure is to scale.
  \item The outer walls of the arena are at least \SI{350}{mm} high, and the
        interior surface is white plastic-coated hardboard.

>   \item Each scoring zone is \SI{2}{m} $\times$ \SI{2}{m} $\pm$ \SI{100}{mm},

Reviewer: ```suggestion
  \item Each scoring zone is \SI{1}{m} $\times$ \SI{1}{m} $\pm$ \SI{100}{mm},
```

\label{spec:arena}

\begin{enumerate}

>   \item The arena floor is an \SI{8.4}{m} $\times$ \SI{8.4}{m} rectangle. The

Reviewer: ```suggestion
  \item The arena floor is an \SI{5.4}{m} $\times$ \SI{5.4}{m} rectangle. The
```

Author: The reason for 5.4 was to match the SR wooden arena. What are we using for this to build the arena?

Reviewer: No, the reason for the original 8.4 was based on truss lengths. The new 5.4m is based on the available truss.

        tolerance of these two dimensions is $\pm$ \SI{250}{mm}.
  \item The floor of the arena is carpeted.
  \item The layout of the arena is given in Figure~\ref{fig:arena}. This
        figure is to scale.
  \item The outer walls of the arena are at least \SI{350}{mm} high, and the
        interior surface is white plastic-coated hardboard.

>   \item Each scoring zone is \SI{2}{m} $\times$ \SI{2}{m} $\pm$ \SI{100}{mm},
>         resulting in a total size of \SI{6}{m} $\times$ \SI{6}{m} $\pm$ \SI{200}{mm}
>         for the nine scoring zones.
>   \item Scoring zones are bounded by tape around the perimeter
>         and internal boundaries on the floor. The inside edge of the tape marks the outside
>         edge of the scoring zone.
>   \item The raised area in the centre of the arena is \SI{2}{m} $\times$ \SI{2}{m} $\pm$ \SI{100}{mm},
>         with a height of ???mm $\pm$ \SI{10}{mm}.

Reviewer: ```suggestion
  \item Each scoring zone is \SI{1}{m} $\times$ \SI{2}{m} $\pm$ \SI{100}{mm},
        resulting in a total size of \SI{3}{m} $\times$ \SI{3}{m} $\pm$ \SI{200}{mm}
        for the nine scoring zones.
  \item Scoring zones are bounded by tape around the perimeter
        and internal boundaries on the floor. The inside edge of the tape marks the outside
        edge of the scoring zone.
```
Correct zone sizes.

  \label{fig:arena}
\end{figure}

\begin{sidewaysfigure}
  \centering
  \includegraphics[scale=0.58]{fig-sidewall.pdf}
  \caption{Layout of markers along each arena wall.}
  \label{fig:sidewall}
\end{sidewaysfigure}

\begin{figure}
  \centering
  \includegraphics[scale=0.5]{fig-markers.pdf}
  \caption{Layout of markers on the arena walls.}
  \label{fig:markers}
\end{figure}

\subsection{Tokens}
\refstepcounter{SpecID}
\label{spec:tokens}

\begin{enumerate}
  \item Tokens are cuboids with side length \SI{100}{mm} $\pm$ \SI{15}{mm}.
  \item The exterior surface of a token has an AprilTag marker printed upon it. The marker is identical on all faces.
  \item All tokens belonging to the same corner will have the same marker ID.
  \item Tokens are arranged as indicated in Figure~\ref{fig:arena}.
\end{enumerate}

\begin{table}[t]
      \centering
      \begin{tabular}{lcc}
            \toprule
            \textbf{Item} & \textbf{Marker Numbers} & \textbf{Marker Size (mm)} \\
            \midrule

>             Arena boundary & {} 0 -- 27 & 250 \\

Author: Issue: We should change these to 200mm

  \label{fig:arena}
\end{figure}

\begin{sidewaysfigure}
  \centering
  \includegraphics[scale=0.58]{fig-sidewall.pdf}
  \caption{Layout of markers along each arena wall.}
  \label{fig:sidewall}
\end{sidewaysfigure}

\begin{figure}
  \centering
  \includegraphics[scale=0.5]{fig-markers.pdf}
  \caption{Layout of markers on the arena walls.}
  \label{fig:markers}
\end{figure}

\subsection{Tokens}
\refstepcounter{SpecID}
\label{spec:tokens}

\begin{enumerate}

>   \item Tokens are cuboids with side length \SI{150}{mm} $\pm$ \SI{25}{mm}.

Reviewer: According to #15 we have 102mm boxes.
